test/units/modules/storage/netapp/test_na_ontap_aggregate.py

```diff
@@ -0,0 +1,215 @@
+""" unit tests for Ansible module: na_ontap_aggregate """
+
+from __future__ import print_function
+import json
+import pytest
+
+from units.compat import unittest
+from units.compat.mock import patch, Mock
+from nose.plugins.skip import SkipTest
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+import ansible.module_utils.netapp as netapp_utils
+
+from ansible.modules.storage.netapp.na_ontap_aggregate \
+    import NetAppOntapAggregate as my_module  # module under test
+
+if not netapp_utils.has_netapp_lib():
+    raise SkipTest("skipping as missing required netapp_lib")
```

Comment:
Needs testing, though I thinnk you want something like:
```suggestion
    pytest.skip("skipping as missing required netapp_lib")
```

Reply:
Unfortunately, pytest3.x broke this syntax.  The new syntax is:
```
    pytestmark = pytest.skip('skipping as missing required netapp_lib')
```

Reply:
Fixed.